fetch.bs

```diff
@@ -5153,6 +5162,8 @@ constructor must run these steps:
    "<code>navigate</code>", then set it to "<code>same-origin</code>".
    <!-- This works because we have reset request's client too. -->
 
+   <li><p>Unset <var>request</var>'s <a for=request>reload-navigation flag</a>.
```

Comment:
Nice! Is this tested?

Reply:
Added a test in https://github.com/w3c/web-platform-tests/pull/10192 (fetch/api/request/request-reset-attributes.https.html)